keps/sig-architecture/3136-beta-apis-off-by-default/README.md

```diff
@@ -0,0 +1,286 @@
+
+
+### Notes/Constraints/Caveats (Optional)
+
+### Risks and Mitigations
+
+Adoption of beta features will slow.
+Given how kubernetes is now treated, this is a good thing, not a bad thing.
+Those users that want to move quickly and get new features can do so by enabling all beta feature
+or just enabling those that are important for their workload.
+The [PRR survey](https://datastudio.google.com/reporting/2e9c7439-202b-48a9-8c57-4459e0d69c8d/page/Cv5HB) shows that 
+over 30% of production clusters have alpha features enabled, so clsuter-admins are willing and able to enable features
+that are not on by default when they are desired.
+
+## Design Details
+
+<!--
+This section should contain enough information that the specifics of your
+change are understandable. This may include API specs (though not always
+required) or even code snippets. If there's any ambiguity about HOW your
+proposal will be implemented, this is the place to discuss them.
+-->
+
+### Test Plan
+
+Integration tests will be written to ensure that no new beta APIs are enabled in the kube-apiserver by default.
+Unit tests will be written to ensure that the new flag functionality works as expected.
+
+### Graduation Criteria
+
+This KEP is a policy KEP, not a feature KEP.  It will start as GA.
+
+#### GA
+
+- Integration and unit tests from above.
+- updating the enablement docs for beta
+  - https://kubernetes.io/docs/reference/using-api/#api-versioning
+  - https://kubernetes.io/docs/reference/command-line-tools-reference/feature-gates/#using-a-feature 
+    Even though that is talking about feature gates, it is likely worth calling out there that new beta REST APIs are no
+    longer enabled by default)
+
+### Upgrade / Downgrade Strategy
+
+The additional command line flag format for `--runtime-config` will not be recognized on older levels of kubernetes.
+This means that when downgrading, cluster-admins will have to adjust their CLI arguments if they opted into a new beta API.
+This is congruent to flag handling for new features today.
+Because this only impacts new beta APIs, there is no behavior change for existing APIs on upgrade.
+
+### Version Skew Strategy
+
+Because this only impacts new beta APIs, there is no novel skew risk.
+
+## Production Readiness Review Questionnaire
+
+Not applicable because this is a policy KEP.
+
+## Implementation History
+
+<!--
+Major milestones in the lifecycle of a KEP should be tracked in this section.
+Major milestones might include:
+- the `Summary` and `Motivation` sections being merged, signaling SIG acceptance
+- the `Proposal` section being merged, signaling agreement on a proposed design
+- the date implementation started
+- the first Kubernetes release where an initial version of the KEP was available
+- the version of Kubernetes where the KEP graduated to general availability
+- when the KEP was retired or superseded
+-->
+
+## Drawbacks
+
+<!--
+Why should this KEP _not_ be implemented?
+-->
```

Comment:
We miss a lot of real feedback about possible incompatibilities between APIs, there are cases that just enabling an API causes problems.
We also reduce the signal to CIs and people enabling betas, that is significant

Reply:
+1 @liggitt also agree @aojea. We increase the burden on ourselves for sure.

Reply:
>  (though that will require CI configuration changes that should be noted in this KEP).

added to criteria.